elife/utils.sls

```diff
@@ -1,25 +1,18 @@
 utils-scripts:
     file.recurse:
-        - name: /usr/local/utils/
+        # this state copies the scripts:
+        # build_vars, retry, set_local_revision, update_python_dependency, wait_for_port
+        # this directory was only available in interactive login shells
+        # it was not available to commands executed using su or sudo
+        # for example, running a cmd.run state with a user other than root
+        # or sudo'ing to root to run a command
+        #- name: /usr/local/utils/
+        # this directory is always on the PATH
+        - name: /usr/local/bin
         - source: salt://elife/utils
         - file_mode: 555
 
 utils-scripts-path:
-    file.managed:
+    file.absent:
         - name: /etc/profile.d/utils-path.sh
         - source: salt://elife/config/etc-profile.d-utils-path.sh
```

Comment:
don't need `source`?

Reply:
you're right, we don't. I couldn't rm that file as well